app/views/curation_concerns/base/_related_files.html.erb

```diff
@@ -14,11 +14,11 @@
       </tr>
     </thead>
     <tbody>
-      <%= render partial: 'member', collection: presenter.file_set_presenters %>
+      <%= render partial: 'member', collection: presenter.member_presenters %>
     </tbody>
   </table>
 </div>
 <% elsif can? :edit, presenter.id %>
-  <h2>Files</h2>
-  <p class="text-center"><em>This <%= presenter.human_readable_type %> has no files associated with it. You can add one using the "Attach a File" button below.</em></p>
+  <h2>Members</h2>
```

Comment:
`Members` makes sense to me in this context, but it could (?) trip up end-users. Thoughts, @projecthydra/sufia-ui-ux-advisors ?

We recently merged a PR in Sufia where we referred to an indeterminate thing that could be either a Work or a Collection as an "item," FWIW.